### DISCUSSION TITLE
Планы по ботам

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#378] [Q&A]
trickerer, у тебя есть планы по внедрению твоих ботов в другие дополнения wow?

**Comment by trickerer** [comment]
Уже отвечал здесь:
https://github.com/trickerer/Trinity-Bots/discussions/361, https://github.com/trickerer/Trinity-Bots/discussions/234

**Reply by xBarselonx** [reply]
Я помню про свой вопрос. Но на всякий случай решил задать вопрос. В некоторый сообществах люди проявляют интерес, например к вов классик.

**Reply by trickerer** [reply]
Ну так во-первых где эти сообщества и сколько этих людей, в сравнении с 3.x.x, например. Во-вторых, на личе я хоть иногда что-то делаю в игре, в предыдущие не играл сто лет. К тому же никакой ностальгии по временам классики у меня нет, так как практически не застал. В БК хотя бы были (мои) лучшие времена вов, а в классике я ценю лишь оригинальные цепочки высокоуровневых квестов, напрочь кастрированные в дополнении, да ту атмосферу реальности прогресса при прокачке и получении шмота, которая так и осталась в промежутке до 60 уровня. С контентом БК с приходом лича была такая же фигня с квестами и требованиями по квестам для доступа в рейды.
В общем, никаких планов на ботов я не составляю. А все свои задумки ориентирую на ту версию в которую люди ещё играют.

**Reply by xBarselonx** [reply]
> > К тому же никакой ностальгии по временам классики у меня нет
> 
> ок. вопрос снимается вопрос с повестки дня. далее я бросил читать. спасибо!

**Reply by xBarselonx** [reply]
> Ну так во-первых где эти сообщества и сколько этих людей

Они все здесь (https://github.com/vmangos/core/issues). А также я дискорд . И им все это интересно, хотя уже прошло немало лет. Например, там есть также боты, неидеальные как у тебя.
